doc/release-notes/6.2-release-notes.md

```diff
@@ -0,0 +1,401 @@
+
+### Assign a Role With Higher Permissions Than Its Own Role Has Been Fixed
+
+The permissions required to assign a role have been fixed. It is no longer possible to assign a role that includes permissions that the assigning user doesn't have.
+
+### Geospatial Metadata Block Fields for North and South Renamed
+
+The Geospatial metadata block fields for north and south were labeled incorrectly as ‘Longitudes,’ as reported on #5645. After updating to this version of Dataverse, users will need to update all the endpoints that used ‘northLongitude’ and ‘southLongitude’ to ‘northLatitude’ and ‘southLatitude,’ respectively.
+
+### OAI-PMH Error Handling Has Been Improved
+
+OAI-PMH error handling has been improved to display a machine-readable error in XML rather than a 500 error with no further information.
+
+> - /oai?foo=bar will show "No argument 'verb' found"
+> - /oai?verb=foo&verb=bar will show "Verb must be singular, given: '[foo, bar]'"
+
+### Granting File Access Without Access Request
+
+A bug introduced with the guestboook-at-request, requests are not deleted when granted, they are now given the state granted.
+
+## 💾 Persistence
+
+### Missing Database Constraints
+
+This release adds two missing database constraints that will assure that the externalvocabularyvalue table only has one entry for each uri and that the oaiset table only has one set for each spec. (In the very unlikely case that your existing database has duplicate entries now, install would fail. This can be checked by running
+
+```
+SELECT uri, count(*) FROM externalvocabularyvalue group by uri;
+```
+And:
+```
+SELECT spec, count(*) FROM oaiset group by spec;
+```
+Then removing any duplicate rows (where count>1).
+
+
+### Universe Field in Variablemetadata Table Changed
+
+Universe field in variablemetadata table was changed from **varchar(255)** to **text**. The change was made to support longer strings in "universe" metadata field, similar to the rest of text fields in variablemetadata table.
+
+### Postgres Versions
+
+This release adds install script support for the new permissions model in Postgres versions 15+, and bumps FlyWay to support Postgres 16.
+
+Postgres 13 remains the version used with automated testing.
+
+
+## 🌐 API 
+
+### Listing Collection/Dataverse API
+
+Listing collection/dataverse role assignments via API still requires ManageDataversePermissions, but listing dataset role assignments via API now requires only ManageDatasetPermissions.
+
+### New API Endpoint for Clearing an Individual Dataset From Solr
+
+A new Index API endpoint has been added allowing an admin to clear an individual dataset from Solr.
+
+### New Accounts Metrics API
+
+Users can retrieve new types of metrics related to user accounts. The new capabilities are [described](https://guides.dataverse.org/en/6.2/api/metrics.html) in the guides.
```

Comment:
It would be nice to link to the specific section on user accounts.

Reply:
It seems it is a parameter so all the guide would be related to this?